CONTRIBUTING.md

```diff
@@ -8,34 +8,39 @@ If you have questions about this document or anything not covered here? Come cha
 
 - All code and doc submissions are done through pull requests against the `devel` branch.
 - Take care to make sure no merge commits are in the submission, and use `git rebase` vs `git merge` for this reason.
-- We ask all of our community members and contributors to adhere to the [Ansible code of conduct](http://docs.ansible.com/ansible/latest/community/code_of_conduct.html). If you have questions, or need assistance, please reach out to our community team at [codeofconduct@ansible.com](mailto:codeofconduct@ansible.com)   
+- We ask all of our community members and contributors to adhere to the [Ansible code of conduct](http://docs.ansible.com/ansible/latest/community/code_of_conduct.html). If you have questions, or need assistance, please reach out to our community team at [codeofconduct@ansible.com](mailto:codeofconduct@ansible.com)
 
 ## Setting up your development environment
 
-In this example we are using [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/), but any virtual environment will do.
+Install `tox` if it is not already installed on your system. This can be done with `pip` or your system package manager.
 
 ```bash
-(host)$ pip install virtualenvwrapper
-(host)$ mkvirtualenv ansible-runner
-(host)$ pip install -e .
+pip install tox -c test/constraints.txt
 ```
 
-When done making changes, run:
+Setup a test virtual environment and activate it:
 
 ```
-(host)$ deactivate
+tox -e dev
+source .tox/dev/bin/activate
```

Comment:
Did you know that tox has a `--devenv` option? It basically allows to create a regular venv out of a testenv that is defined in `tox.ini`.

So you'd use this as
```suggestion
tox -e dev --devenv dev-venv
. dev-venv/bin/activate
```

No need to rely on internal tox's envs. Plus, you could get an env for things that are more specific to what you need, like
```console
$ tox -e py38-ansible29 --devenv ans29-venv
```

refs:
* https://tox.readthedocs.io/en/latest/config.html#cmdoption-tox-devenv
* https://tox.readthedocs.io/en/latest/changelog.html#v3-13-0-2019-06-24